examples/esp_idf/hello/pytest/test_sample.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import pytest
+import time
+import yaml
+import datetime
+
+LOGGER = logging.getLogger(__name__)
+
+pytestmark = pytest.mark.anyio
+
+async def test_hello(board, device, credentials_file):
+
+    # Read credentials
+    with open(credentials_file, 'r') as f:
```

Comment:
We aren't using the credentials file for anything in this function, so we can remove it. The `pytest-hil` plugin uses it to get and load the wifi credentials.